_docs/principles/accessibility.md

```diff
@@ -198,61 +294,71 @@ Helix minimum spacing standards require: 
+### Write meaningful alternative text for images
 
-#### Focus for Input Field
+Screen readers read the text you enter in the `alt` attribute that you associate with an image. If an image provides important contextual information, then use the `alt` attribute to describe what is happening in the picture. If you do not use the `alt` attribute, some screen readers read the name of the image file, which introduces unnecessary confusion and results in a poor user experience.
 
-In this example, we provide a clear visual feedback that the focus has moved to the input field by changing the border color and adding a highlight to give the user clear indication of where they’re currently taking action.
+If an image is purely decorative, use `aria-hidden="true"`. Do not use an empty `alt` attribute.
+
+The `aria-hidden="true"` also applies to inline SVG images such as icons. Inline SVGs require that you to add other elements, for example `<title>` and `<description>`, to make them accessible. For more information about SVGs, see [Accessible SVGs](https://css-tricks.com/accessible-svgs/).
 
 {% endcolumn %}
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+</div>
 
-{% figure [caption:"Default state"][class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-3.svg"/>
-{% endfigure %}
+</section>
 
-{% endcolumn %}
+<section class="static-section"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+<div class="hxRow"  markdown="1">
 
-{% figure [caption:"Focused state"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-4.svg"/>
-{% endfigure %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Text guidelines
+
+Consider the following text guidelines when you construct an accessible UI.
+
+### Be succinct
+
+Write short labels and be concise when writing helper text. Listening to lengthy field names and links can be cumbersome for users that use screen readers. The shorter the label, the more quickly a screen reader can work through the content.
+
+### Identify required inputs
+
+Use the HTML5 `required` attribute to distinguish between required and optional inputs. Screen readers cannot interpret an asterisk (``*``), which is the convention used to visually identify required inputs.
+
+### Link text should convey what happens when clicked
+
+Sentences that include links should include a text description before the link and then a unique name for the link. Do not use **Click Here** as a link naming convention. For example, use 'Visit the [Professional Services](https://www.rackspace.com/professional-services) page to learn more about how we can help transform your IT infrastructure'. Do not use 'Click [here](https://www.rackspace.com/professional-services) to learn more about how we can help transform your IT infrastructure'.
```

Comment:
Is there any need to make the links in the examples actual links? I would remove those.